Makefile

```diff
@@ -179,11 +181,17 @@ endif
 # Generate bundle manifests and metadata, then validate generated files.
 .PHONY: bundle
 bundle: manifests kustomize
+ifneq ($(and $(REPLACES),$(OPERATOR_NAMESPACE)),)
 	operator-sdk generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
+	cd config/manifests/bases && sed -e "s/@REPLACES@/$(REPLACES)/" -e "s/@VERSION@/$(VERSION)/" kustomization-base.yaml > kustomization.yaml
+	cd config/manifests && $(KUSTOMIZE) edit set namespace $(OPERATOR_NAMESPACE)
```

Comment:
Why is this needed? NAMESPACE is already taken care of in `config/default/kustomization.yaml` and I would like to have a single source of truth regarding the namespace.

I think it will be useful to use this command to set the namespace that is already defined there.

Reply:
removed, not required.